english/explore-outside/talk-child/talk-more/nice-talk/help-child/abandon-hope/toss-keys/toss-keys.md

```diff
@@ -0,0 +1 @@
+You aim for the heart of the lake and lob the keys. They land with an awkward plop and sink, as keys do.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.